build.sbt

```diff
@@ -168,9 +169,39 @@ lazy val `i18n-frontend` = project.in(file("i18n/frontend"))
     jsDependencies += RuntimeDOM % Test
   )
 
+lazy val `css-macros` = project.in(file("css/macros"))
+  .settings(commonSettings: _*)
+  .settings(
+    libraryDependencies ++= Seq("org.scala-lang" % "scala-reflect" % scalaVersion.value),
+    libraryDependencies ++= cssMacroDeps.value
+  )
+
+lazy val `css-shared` = crossProject.crossType(CrossType.Pure).in(file("css/shared"))
```

Comment:
We might not need `udash-core` dependencies and provide a nice scalacss flavour for current users. This could be a gateway drug to Udash :D